README.md

```diff
@@ -11,202 +11,273 @@ The goal of the project is to create a common HTML5 UI abstraction layer for the
 
 It supports two-way JavaScript bindings (to call JavaScript from C/C++/Go and to call C/C++/Go from JavaScript).
 
-It uses Cocoa/WebKit on macOS, gtk-webkit2 on Linux and Edge on Windows 10.
+## Platform Support
 
-## [API Documentation](https://webview.dev "API Documentation")
+Platform | Technologies
+-------- | ------------
+Linux    | [GTK 3][gtk], [WebKitGTK][webkitgtk]
+macOS    | Cocoa, [WebKit][webkit]
+Windows  | [Windows API][win32-api], [WebView2][ms-webview2]
 
-API Documentation is available online for this repository and all available bindings at https://webview.dev.
+## Documentation
 
-Contributions to the documentation are managed at https://github.com/webview/docs.
+We have started working on publishing documentation at [webview.dev] but you can always find the most up-to-date documentation right in the source code. Improving the documentation is a continuous effort and you are more than welcome to [offer suggestions][issues-new] or [contribute with content][docs-repo]. Please bear with us if the latest updates are not yet published.
 
-## Bindings
+## Prerequisites
+
+Your compiler must support minimum C++11 except for platforms that require a more modern version.
+
+### Linux
+
+The [GTK][gtk] and [WebKit2GTK][webkitgtk] libraries are required for development and distribution. You need to check your Linux distribution regarding how to install those those.
+
+Debian-based systems:
+
+* Packages:
+  * Development: `apt install libgtk-3-dev libwebkit2gtk-4.0-dev`
+  * Production: `apt install libgtk-3-0 libwebkit2gtk-4.0-37`
 
-This repository contains bindings for C, C++, and Go. Bindings for other languages are maintained separately.
+BSD-based systems:
```

Comment:
BSD-based systems aren't Linux. It's a bit awkward for them to be listed under the Linux heading. Maybe "Linux" could be changed to something more general or a new BSD section could be added.

Reply:
Good point!

Reply:
Please see if the fixed text is appropriate.

Reply:
Perhaps one problem with the new wording is that macOS is also Unix-like.

Reply:
macOS is also Unix-like. Maybe try "Linux and BSD". If so, "Linux distribution" needs to be changed to "package repositories" or something more generic.

Reply:
I like "package repositories" and "Linux and BSD" solves other issue.

Reply:
Given that using "Unix-like" was a problem because macOS is also Unix-like, do you think that there is still an issue with the new wording because macOS is based on BSD?

Reply:
Technically speaking, this may still be an issue, but I don't think there would be any actual confusion. I'm wracking my brain for another way to phrase it and can't think of anything much clearer.